server/verify.go

```diff
@@ -44,6 +47,24 @@ type VerifyOpts struct {
 	IntermediateCerts []*x509.Certificate
 }
 
+type gceSecurityProperties struct {
+	SecurityVersion             int64 `asn1:"optional"`
+	IsProduction                bool  `asn1:"optional"`
```

Comment:
This field and the ones below are internal interfaces, should we have them in this library or call them reserved and provide a way to extract them from the library?

Reply:
I don't have context on the security implications but in this library we're only interested in IsProduction and perhaps SecurityVersion in the future. I should be able to remove the other fields from the struct (so they won't be unmarshaled) and only use the ones we need?

Reply:
Let's just omit the last 4 fields here (as they are optional anyway, and aren't really something customers can control).

So we would just have:
```go
type gceSecurityProperties struct {
	SecurityVersion             int64 `asn1:"optional"`
	IsProduction                bool  `asn1:"optional"`
}
```